include/sys/signalfd.h

```diff
@@ -0,0 +1,67 @@
+///////////////////////////////////////////////////////////////////////////////
+// Licensed Materials - Property of IBM
+// ZOSLIB
+// (C) Copyright IBM Corp. 2022. All Rights Reserved.
+// US Government Users Restricted Rights - Use, duplication
+// or disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
+///////////////////////////////////////////////////////////////////////////////
+
+#ifndef ZOS_SYS_SIGNALFD_H
+#define ZOS_SYS_SIGNALFD_H
+
+#include <stdint.h>
+#include <signal.h>
+
+/* Flags for signalfd.  */
+enum
+  {
+    SFD_CLOEXEC = 02000000,
+#define SFD_CLOEXEC SFD_CLOEXEC
+    SFD_NONBLOCK = 00004000
+#define SFD_NONBLOCK SFD_NONBLOCK
+  };
+
+
+struct signalfd_siginfo {
+	uint32_t ssi_signo;
```

Comment:
Use two spaces for indentation like https://github.com/ibmruntimes/zoslib/blob/main/src/zos-mount.c

Reply:
Done